Python/ceval.c

```diff
@@ -873,7 +873,8 @@ match_keys(PyThreadState *tstate, PyObject *map, PyObject *keys)
             }
             goto fail;
         }
-        PyObject *value = PyObject_CallFunctionObjArgs(get, key, dummy, NULL);
+        PyObject *args[] = { key, dummy };
+        PyObject *value = PyObject_Vectorcall(get, args, 2, NULL);
```

Comment:
Just replacing PyObject_CallFunctionObjArgs shows a 2% performance enhancement on the micro benchmark.